lib/opensusebasetest.pm

```diff
@@ -54,6 +55,54 @@ sub export_logs {
     save_screenshot;
 }
 
+sub export_wicked_logs {
+    my $self = shift;
+
```

Comment:
@nilxam  Yes, thank you
I added it to export logs, as the wicked part is executed only if network is down

http://10.100.98.90/tests/1913